app/assets/scripts/components/common/feeback-form.js

```diff
@@ -1,22 +1,17 @@
 import React, { useContext, useState } from 'react';
-import { PanelBlockFooter } from '../common/panel-block';
 import { withRouter } from 'react-router-dom';
-import Dropdown, { DropTitle } from './dropdown';
+import Dropdown from './dropdown';
 import Button from '../../styles/button/button';
 import SubmitIssueTray from '../explore/submit-issue-tray';
 import styled from 'styled-components';
+import { tint } from 'polished';
+import { stylizeFunction, themeVal } from '../../styles/utils/general';
 
-const ButtonDiv = styled.span`
-  width: 3rem;
-  height: 3rem;
-  button {
-    width: 3rem;
-    height: 3rem;
-  }
-  button::before {
-    margin-top: -5px;
-    margin-left: -5px;
-  }
```

Comment:
@akshay2408 why do you remove this? It will make the feedback icon not centered.

Reply:
![image](https://github.com/worldbank/WB-rezoning-explorer/assets/1979569/d89f9710-95b1-46b4-a784-63b77e24c8d2)